gitlab/utils.py

```diff
@@ -64,25 +68,38 @@ def _transform_types(
     FileAttribute types with tuples to prepare fields for requests' MultipartEncoder:
     https://toolbelt.readthedocs.io/en/latest/user.html#multipart-form-data-encoder
 
+    ``is_params``: data will be sent to ``requests`` in the ``params`` field.
```

Comment:
I think the point here is that we prepare the data for that, not that we actually send it to params, right?
E.g. looking at the sentence above it and my comment before, it might go something like:
```suggestion
    If ``transform_data`` is ``True`` (default), also transforms the ``data`` dict 
    with fields suitable for encoding as query parameters for GitLab's API:
    https://docs.gitlab.com/ee/api/#encoding-api-parameters-of-array-and-hash-types
```

Reply:
Done